operations/research-and-development/engineering/team/README.md

```diff
@@ -61,3 +38,43 @@ Below is how SET prioritizes what is worked on.
    * Pick up any bugs in triage without an obvious owning feature team.
    * Work on bugs in this order: hotfix, release, customer reported, other.
 4. [~Ask R&D](https://community-daily.mattermost.com/core/channels/ask-r-and-d) channel — this channel should be monitored, but checking it once or twice per day is sufficient.
+
+## Sources of escalations
+
+Work for SET comes from various sources:
+
+1. OpsGenie: Support has an e-mail address they will send urgent escalations to. These result in pages in OpsDuty, first sent to the SET Primary, then to SET Back-up and ultimately to the SET Lead when not timely acknowledged.
+2. The [~Sustained Engineering](https://community-daily.mattermost.com/core/channels/sustained-engineering) channel. Support may use this channel to ask questions or raise urgent issues.
+3. The SET [Kanban JIRA board](https://mattermost.atlassian.net/secure/RapidBoard.jspa?rapidView=33). Tickets in the TO DO column should be organized from highest priority to lowest priority, based on the [Priority of Work](./#priority-of-work) section. The SET lead should make sure that the tickets are priority ordered but team members can move tickets around to meet the correct priorities as necessary. If tickets get stuck in a certain state, it's also the SET responsibility to push them to completion, either by nudging people or taking them over.
+4. The [~Ask R&D](https://community-daily.mattermost.com/core/channels/ask-r-and-d) channel. This is the place where support and community can ask any type of R&D related questions.
+
+## SET Weekly Cycle
+We use a Mattermost Playbook to run the SET cycle. For this, the SET Lead creates a new run of the [SET Cycle](https://community-daily.mattermost.com/playbooks/playbooks/w36ro78cgj8qtjq5mf6say7bxe/preview) playbook and follows the steps as codified by the playbook. Overall a SET cycles consists of three phases:
+
+1. Start: review all ongoing issues and make sure they have appropriate owners in the current SET team.
+2. Monitor: Monitor the sources of SET escalations (previous section) and act accordingly (see next section)
+3. Hand-over: by Friday end of day, make sure the next week's SET team (especially the SET Lead) is aware of ongoing issues and have sufficient context to continue the work. Specifically:
```

Comment:
@agarciamontoro what do you think about this?

Reply:
That's way better, thanks! One thing I would add: who is the owner of this hand-over? Should the dev be in charge of communicating that they are leaving SET and unassigning them from the tickets? Should the lead make sure that all tickets are handed over?

Reply:
SET Lead I'd say, I'll add that.